frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue

```diff
@@ -324,6 +442,18 @@ export default {
     allLoadBalancerClassNames () {
       return this.loadBalancerClassNamesByCloudProfileName(this.cloudProfileName)
     },
+    partitionIDs () {
+      return this.partitionIDsByCloudProfileNameAndRegion({ cloudProfileName: this.cloudProfileName, region: this.region })
+    },
+    firewallImages () {
+      return this.firewallImagesByCloudProfileName(this.cloudProfileName)
+    },
+    firewallSizes () {
+      return map(this.firewallSizesByCloudProfileNameAndRegionAndZones({ cloudProfileName: this.cloudProfileName, region: this.region, zones: [ this.partitionID ] }), 'name')
```

Comment:
```suggestion
      const cloudProfileName = this.cloudProfileName
      const region = this.region
      const zones = [this.partitionID]
      const firewallSizes = this.firewallSizesByCloudProfileNameAndRegionAndZones({ cloudProfileName, region, zones })
      return map(firewallSizes, 'name')
```
suggestion not checked for linting / correctness